public/app/percona/backup/components/StorageLocations/StorageLocations.test.tsx

```diff
@@ -64,6 +68,7 @@ describe('StorageLocations', () => {
   });
 
   it('should open the modal by clicking the "Add" button', async () => {
+    jest.spyOn(console, 'error').mockImplementation();
```

Comment:
why are we silencing the error here?

Reply:
There was some weird error that occurred even though I didn't change anything in this test, so I put it there and didn't notice it after in review I guess. Error still occurs for some reason so I fixed it.